src/mock_crust_detail/test_node.rs

```diff
@@ -153,10 +153,7 @@ pub fn create_nodes(network: &Network,
                                  use_cache));
         let _ = poll::nodes(&mut nodes);
     }
-
-    // (Last node consumes the config objects.)
-    nodes.push(TestNode::new(network, Some(crust_config), config, false, use_cache));
-    let _ = poll::nodes(&mut nodes);
+    drop(config);
```

Comment:
What is the point of this call? The value gets dropped anyway, when it reaches the end of the scope.